homeassistant/components/jewish_calendar/service.py

```diff
@@ -0,0 +1,53 @@
+"""Services for Jewish Calendar."""
+
+import datetime
+
+from hdate import HebrewDate
+from hdate.omer import Nusach, Omer
+import voluptuous as vol
+
+from homeassistant.core import (
+    HomeAssistant,
+    ServiceCall,
+    ServiceResponse,
+    SupportsResponse,
+)
+
+from . import JewishCalendarConfigEntry
+from .const import DOMAIN
+
+OMER_SCHEMA = vol.Schema(
+    {
+        vol.Required("date", default=datetime.date.today): datetime.date,
+        vol.Required("nusach", default="sfarad"): str,
```

Comment:
What are the possible options?

Reply:
I have an enum, how should I specify that it should be one of the enum values?

Reply:
`vol.In` I believe